web/pageComponents/shared/Hero/FiftyFiftyHero.tsx

```diff
@@ -70,11 +69,15 @@ const HeroActionLink = ({ action, ...rest }: { action: LinkData }) => {
   if (action.type === 'internalUrl') {
     const linkLocale = getLocaleFromName(action.link?.lang)
     return (
-      <NextLink href={url} locale={Flags.IS_DEV ? linkLocale : undefined} passHref legacyBehavior>
-        <Link variant="readMore" aria-label={ariaLabel} {...rest}>
-          {action.label}
-        </Link>
-      </NextLink>
+      <Link
+        href={url}
+        locale={Flags.IS_DEV ? linkLocale : undefined}
```

Comment:
almost 200% sure this is if condition is a "missed" feature flag from our old setup. Locale should be always linkLocale, doesn't make sense to have it as undefined here.